notebooks/examples/python/QuickstartNotebook.py

```diff
@@ -140,18 +174,35 @@
 # COMMAND ----------
 
 tripsWithIndex = (trips
-  .withColumn("pickup_h3", point_index(col("pickup_geom"), lit(9)))
-  .withColumn("dropoff_h3", point_index(col("dropoff_geom"), lit(9)))
+  .withColumn("pickup_h3", mos.point_index_geom(col("pickup_geom"), lit(optimal_resolution)))
```

Comment:
@edurdevic love the reference to a variable instead of literal I did before! way better this way!